---
title: "Gene expression database guide"
author: Erick Lu
output:
  html_document:
    toc: true
---

# Tissue Expression Databases

In biomedical science, one of the most commonly asked questions is: where is a gene expressed? In what tissues, and in what cell types? This information is critical when assessing the probability that therapeutics that target a specific protein or pathway will have undesireable or off-target effects. Luckily, several publicly available databases can be used determine the overall expression pattern for a gene of interest. Here, I list some databases that I have found to be useful.

## biogps.com

For human tissue-level and cell type-level expression, http://biogps.org/ is a great place to start. Entering the name of your gene in the search box:

should show results for the different species available within biopgps:

Selecting the species you are interested in will show the gene expression patterns for the GeneAtlas U133A, gcrma by default. We can change the dataset by clicking on the change dataset button in the top right of the center panel:

I typically check Default Datasets -> Primary Cell Atlas.

This provides a nice overview of the cell types that have expression of your gene of interest.

## gtexportal.org

Another database that gives tissue-level expression (i.e. which organs have high/low expression of your gene) is the gTEX Portal: https://gtexportal.org/home/. Simply type in the name of the gene and a plot will show up showing overall expression of the gene in various whole tissue / organs.

## proteinatlas.org

The human protein atlas contains a huge immunohistochemistry database, in which healthy tissues as well as tumor-bearing tissues have been stained for proteins of interest. Searching for your protein of interest shows the relative expression of the protein across different tissues of the body. Clicking on the tissue will show the actual immunohistochemistry images, so that you can assess whether the staining looks on-target or off-target, as well as determine what the spatial expression pattern of your protein is. This tool is great for comparing the results from the other gene expression databases with. We would expect taht the same tissues where we see high gene expression should also have high expression at the protein level.

The human protein atlas also has a cell line atlas, in which it has profiled by RNA-seq many cancer cell lines. It categorizes the cell lines based on tissue of origin, and allows you to search for the expression level of your gene of interest:

This is useful when you are interested in using cell line models for your project, and need to select a line that is positive or negative for your gene of interest.

## cancer cell line encyclopedia: https://portals.broadinstitute.org/ccle/about

## immgen.org

Immgen is a great database for exploring expression of genes in mouse tissues and cell lines. This website has both a microarray database of sorted cell types as well as an RNA-seq database of sorted cell types. The main focus is on immunology-associated cell subsets.

## 3. Single-cell gene expression databases

There are a few good single-cell RNA-seq databases. Single cell RNA-seq is useful for determining the expression of a gene within a very heterogenous mixture of cells. This is particularly useful for determining the specificity of a gene of interest for a cell type that you wish to target.

### EBI sc database: https://www.ebi.ac.uk/gxa/sc/home

### broad institute sc database: https://singlecell.broadinstitute.org/single_cell

# Human tumor expression databases

In oncology, we also may be interested in how expression levels of a protein within tumors correlate with patient survival. There are databases available to assess these correlations as well.

## firebrowse.org

TCGA expression data across indications, tumor vs normal
http://firebrowse.org/ is a webpage created by the BROAD Institute that allows users to view the expression level of a gene of interest across all the indications within TCGA. For example, if I am interested in GPR183, I would input this into the search bar and the following result will be displayed:


As we can see, each cancer indication is listed on the x-axis, and the expression of the gene is listed on the y-axis. For each indication, the expression of the gene within the tumor tissue samples (red) as well as the expression within the adjacent normal tissue samples (blue) are plotted. The indications are by default alphabetically sorted, but we can also sort the indications based on expression level, as such:


This is a useful way to quickly identify indications that have high expression of your gene, as well as identify indications in which the gene is enriched in tumor tissue compared to the adjacent normal tissue.

### TCGA survival associations

There are multiple websites out there that are useful for analyzing survival associations within TCGA data. Below are three that I have found useful:

### oncolnc.org

cox regression tool
At http://www.oncolnc.org, we can input our gene of interest and it will perform a cox regression analysis on all the indications within TCGA. For example, inputting GPR183 results in the following output, sorted by cox coefficient:

http://www.oncolnc.org/search_results/?q=gpr183

This tells us which cancers are likely to have a positive or negative association with survival. To further determine the survival associations are significant, we need to perform a split in which we determine which tumors have "high" or "low" expression. By pressing the "Plot Kaplan" button, and then performing a simple median split (entering 50:50) into the resulting webpage, we observe that SKCM has a clear survival association in which patients with higher GPR183 on average have higher survival than patietns with lower GPR183. The P-value is also quite small at 0.00817.

#### Gepia2

A more complex tool that allows for more customization than the above is Gepia2, at http://gepia2.cancer-pku.cn/#index. This web tool has many different functionalities. 

To perform a survival analysis, navigate to the survival tab: http://gepia2.cancer-pku.cn/#survival. Then, input the gene of interest, and select the indications you would like to perform the analysis on. You can also customize the type of split, error bars, etc. This tool is a bit more versatile than the tool at oncolnc.org.

This website also lets you export lists of top correlated genes. For example, the genes that are most correlated with GPR183 expression in X are X:


#### PROGGENE

Aside from TCGA, there are a vast number of gene expression studies associated with clinical data that are published in stand-alone journal articles. For example, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE42568 provides gene expression information associated with 104 breast cancer biopsies, which can be associated with the survival outcomes from the patients from which the biopsies were taken.

The web tool at http://genomics.jefferson.edu/proggene/ contains a compilation of a large number of these studies (as well as TCGA). Using this tool, we can input a signature composed of multiple genes as well. This will compute an average expression value using the aggregate expression of all the genes entered, and then assess survival based on your splitting criteria. This feature is nice if you are looking at things such as t cell exhaustion, which is a phenotypic state composed of multiple genes.

### Downloading TCGA data

analyzing the raw TCGA data: short guide - http://www.omnesres.com/tools/tcga/

The raw TCGA data can be obtained from http://gdac.broadinstitute.org/. 

Raw data for TCGA can also be obtained at

https://xenabrowser.net/

# Conclusion

Given that the gene expresion information is collected by different scientists across the world, using different experimental techniques (Microarray, RNA-seq, etc.), it is important to compare and contrast the information provided by multiple databases. If we observe corroborating evidence from multiple databases, we can be more confident about making conclusions about the gene's expression patterns. Ultimately, the results obtained from these databases should be cross-referenced with publications on the gene.
